manifests/s/StefHeyenrath/GitHubReleaseNotes/1.0.9.0/StefHeyenrath.GitHubReleaseNotes.installer.yaml

```
@@ -0,0 +1,16 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: StefHeyenrath.GitHubReleaseNotes
+PackageVersion: 1.0.9.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: GitHubReleaseNotes.exe
+ProductCode: '{32249977-C6C1-40ED-B6A8-7084E69A0E7A}'
+ReleaseDate: 2024-02-02

```

```suggestion
ReleaseDate: 2024-02-02
AppsAndFeaturesEntries:
- UpgradeCode: '{CA1F74E0-DFCA-4774-AE65-9421B4D08FF2}'
```
Hi @StefH,

This is only a suggestion. Other manifest versions have these lines - I'm still learning about this section and I believe they help with version matching.